ui/app/pages/create-account/connect-custodial/trustvault/email/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { PureComponent } from 'react'
+const PropTypes = require('prop-types')
+const { connect } = require('react-redux')
+const actions = require('../../../../../store/actions')
+const EmailScreen = require('./email-screen')
+
+class ConnectTrustVaultEmailForm extends PureComponent {
+  constructor (props) {
+    super(props)
+    this.state = {
```

Comment:
Can we move this `state` field onto the class and remove the (resulting) empty constructor?